docs/content/guides/primer_truncate.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Moving Away From `Primer::Truncate`
+---
+
+This guide will show you how to upgrade from the now-deprecated
+[`Primer::Truncate`](https://primer.style/view-components/components/truncate)
+to the latest
+[`Primer::Beta::Truncate`](https://primer.style/view-components/components/beta/truncate)
+component.
+
+The new `Truncate` component additionally now includes a text slot used for the
+truncated text.
+
+## Some Migration Examples
+
+Migrating the most common use cases of the `Truncate` component simply requires
+changing the name.
+
+Previously we might have something like:
+
+```ruby
+<%= render(Primer::Truncate.new(tag: :p)) { "Some really, really verbose content" } %>
+```
+
+That's now:
+
+```ruby
+<%= render(Primer::Beta::Truncate.new(tag: :p)) { "Some really, really verbose content" } %>
+```
+
+The `tag` argument now defaults to `:span` instead of `:div`, so if we want to
+continue wrapping content in `<div>` tags we'll need to explicitly set that. So,
+if we'd been using:
+
+```ruby
+<%= render(Primer::Truncate.new) { "Some more very very long text" } %>
+```
+
+We can now equivalently write:
+
+```ruby
+<%= render(Primer::Beta::Truncate.new(tag: :div)) { "Some more very very long text" } %>
+```
+
+## Arguments
+
+The following arguments for the component initializer have changed between the deprecated and newer versions
+of the `Truncate` component.
+
+| From `Primer::Truncate` | To `Primer::Beta::Truncate` | Notes                                                                    |
+|-------------------------+-----------------------------+--------------------------------------------------------------------------|
```

Comment:
The markdown parser might be angry about the `+` here?

https://primer-075f5335f7-26611710.drafts.github.io/guides/primer_truncate

<img width="946" alt="image" src="https://user-images.githubusercontent.com/54012/224439267-c48f109d-4f51-432d-8035-2311462e1811.png">

Reply:
Oof, I've been writing in Org-mode too long. Fixed!